File: app-java/src/main/java/com/paytpv/appjava/utils/UserHandler.java
package com.paytpv.appjava.utils;

import com.paytpv.androidsdk.Model.Basic.PTPVUser;

import java.util.ArrayList;

public class UserHandler {
    private static UserHandler singleton = new UserHandler( );
    private ArrayList<PTPVUser> userArray = new ArrayList<>();

    private UserHandler() { }

    public static UserHandler getInstance( ) {
        return singleton;
    }

    public ArrayList<PTPVUser> getUserArray() {
        return userArray;
    }
}

File: app-kotlin/src/main/java/com/paytpv/paytpv/utils/UserHandler.kt
package com.paytpv.paytpv.utils

import com.paytpv.androidsdk.Model.Basic.PTPVUser

class UserHandler {

    private object Holder {
        val INSTANCE = UserHandler()
    }

    companion object {
        val instance: UserHandler by lazy { Holder.INSTANCE }
    }

    var userArray: ArrayList<PTPVUser>? = ArrayList()
}